README.md

```diff
@@ -245,12 +252,6 @@ Injector:
         'hello': 'Something'
 ```
 
-### Extra performance
-
-For even more performance from global caching, a new `main.php` file is provided which ensures database connections aren't made when a cache is available for the current request.
```

Comment:
This has been removed for now. SilverStripe’s authentication middleware (which populates `Security::getCurrentUser()`) makes a database query anyway, so I don’t think we can save anything there. That aside, the “boot” process is much faster now anyway